### PR TITLE
feat: #4407 v3 universal provider

### PR DESCRIPTION
**Pull request by bashleigh**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #4407

### CODE REVIEW

File: packages/elements/src/hooks/use-media-query/media.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    </MediaStateProvider>
+  </Story>
+</Canvas>
+
+Variable | breakpoint sum 
+--- | ---
+`isMobile` | < 768
+`isTable` | >= 768 && < 1024
+`isDesktop` | >= 1024 && < 1440
+`isWideScreen` | >= 1440 && < 1920
```
Comment (bashleigh) [inline packages/elements/src/hooks/use-media-query/media.stories.mdx:38]: defo made a mistake here but can't put my finger on it

File: packages/elements/src/hooks/use-media-query/index.tsx [hunk 2]
```diff
@@ -5,11 +5,16 @@ export interface MediaType {
...
   isWideScreen: boolean
+  isSuperWideScreen: boolean
+  is4KScreen: boolean
 }
 
 export const MOBILE_BREAKPOINT = 768
 export const TABLET_BREAKPOINT = 1024
-export const DESKTOP_BREAKPOINT = 1400
+export const DESKTOP_BREAKPOINT = 1024
+export const WIDESCREEN_BREAKPOINT = 1440
```
Comment (bashleigh) [inline packages/elements/src/hooks/use-media-query/index.tsx:15]: defo made a mistake here but can't put my finger on it

File: packages/elements/src/hooks/use-media-query/media.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+export const AllBreakPointExample = () => {
+  const { isMobile, isTablet, isDesktop, isWideScreen, isSuperWideScreen, is4KScreen, } = useMediaQuery()
+
+  if (isMobile) {
+    return <h1>Is Mobile</h1>
+  }
+
+  else if (isTablet) {
```
Comment (willmcvay) [inline packages/elements/src/hooks/use-media-query/media.stories.tsx:17]: You don't need an else here if you are returning at each conditional

**Review by willmcvay — APPROVED** [review]
One small thing, happy to merge though, not a biggie

File: packages/elements/src/hooks/use-media-query/index.tsx (hunk 2 above)
Comment (bashleigh) [reply packages/elements/src/hooks/use-media-query/index.tsx]: I corrected this, I only needed 5 breakpoints for the 6 conditions

**Pull request merged**